dbld/images/ubuntu-bionic.dockerfile

```diff
@@ -20,8 +20,6 @@ RUN /dbld/builddeps install_apt_packages
 RUN /dbld/builddeps install_debian_build_deps
 RUN /dbld/builddeps install_pip_packages
 
-RUN /dbld/builddeps install_criterion
```

Comment:
Now almost all Ubuntu images use the OBS-packaged criterion version, except for ubuntu-focal.
Can we add an OBS-packaged Criterion version to focal as well? devshell is going to be focal-based soon, it would be nice to use the same version there too.

(Debian images use the non-packaged version)